examples/Training/java/src/training/ReadData.java

```diff
@@ -265,6 +271,33 @@ private void loadPlate() throws Exception {
         }
     }
 
+    /**
+     * Load the MapAnnotations of the currently logged in user.
+     */
+    private void loadMapAnnotations(long imageId) throws Exception {
+        BrowseFacility browse = gateway.getFacility(BrowseFacility.class);
+        ImageData image = browse.getImage(ctx, imageId);
+
+        // load only this user's annotations
+        List<Long> userIds = new ArrayList<Long>();
+        userIds.add(this.user.getId());
+
+        // load only MapAnnotations
+        List<Class<? extends AnnotationData>> types = new ArrayList<Class<? extends AnnotationData>>();
+        types.add(MapAnnotationData.class);
+
+        MetadataFacility metadata = gateway.getFacility(MetadataFacility.class);
+        List<AnnotationData> annotations = metadata.getAnnotations(ctx, image,
+                types, userIds);
+        for (AnnotationData annotation : annotations) {
+            MapAnnotationData mapAnnotation = (MapAnnotationData) annotation;
+            List<NamedValue> list = (List<NamedValue>) mapAnnotation
```

Comment:
It looks as if `AnnotationData.getContent()` could do with some use of generics but that may technically be a breaking change. (Outwith this PR.)